spec/index.html

```diff
@@ -52,28 +48,34 @@
 <body>
 <section id="abstract">
   <p>The Resource Description Framework (RDF) is a framework for
-    representing information in the Web. This document defines an abstract syntax
-    (a data model) which serves to link all RDF-based languages and
-    specifications. The abstract syntax has two key data structures:
-    RDF graphs are sets of subject-predicate-object triples,
-    where the elements may be IRIs, blank nodes, or datatyped literals. They
-    are used to express descriptions of resources. RDF datasets are used
-    to organize collections of RDF graphs, and comprise a default graph
-    and zero or more named graphs.
-    [[[RDF12-CONCEPTS]]] also introduces key concepts and terminology, and discusses
+    representing information in the Web.
+    This document defines an abstract syntax (a data model)
+    which serves to link all RDF-based languages and specifications.
+    The abstract syntax has two key data structures:</p>
+  <ul>
+    <li>RDF graphs are sets of subject-predicate-object triples,
+      where the elements may be IRIs, blank nodes, or datatyped literals.
+      They are used to express descriptions of resources.</li>
+    <li>RDF datasets are used to organize collections of RDF graphs,
+      and comprise a default graph and zero or more named graphs.</li>
```

Comment:
```suggestion
      are comprised of a default graph and zero or more named graphs.</li>
```
This reads better (IMO) when changing to a bullet point list.
(this may not the the right PR to make that change)

Reply:
It's entirely editorial, and I'm erring on the side of having fewer PRs to look at, given the large number of repos to manage.

I also want to see if I can figure out the Echidna/ReSpec timeout issue, which only affects the PR, and not the main branch, before moving on to more changes here and elsewhere.